project.yaml

```diff
@@ -1545,14 +1589,14 @@ Layer:
                   END)
                   *
                   (CASE
-                    WHEN (capital = 'yes') THEN 3
-                    WHEN (capital = '4') THEN 2
+                    WHEN (capital = '4' OR (capital = 'yes' AND admin_level = '4')) THEN 2
                     ELSE 1
                   END)
                 ) AS score
               FROM planet_osm_point
               WHERE place IN ('city', 'town')
                 AND name IS NOT NULL
+                AND capital IS NULL OR capital != '2' OR (capital = 'yes' AND admin_level != '2')
```

Comment:
Is this the counter-part for the capital-names SQL querry?

At capital-names, there is:

```
 WHERE capital = 'yes'
            AND admin_level = '2'
```

So I guess the counter-part should be
```
AND capital IS NULL OR capital != 'yes' OR (capital = 'yes' AND admin_level != '2')
```

???